checker/tests/nullness/Issue880.java

```diff
@@ -0,0 +1,22 @@
+// Test case for Issue 880:
```

Comment:
#880 is a pull request not an issue. The changes reverted #880 (namely PR #876) were trying to enable storing annotations in extends/implements clauses and that is what this test is testing.  But this PR doesn't enable storing annotations in extends/implements clauses.  So I'm confused as to what this is suppose to be testing.  

Also is this test just a duplicate of the tests in https://github.com/typetools/checker-framework/tree/master/checker/jtreg/nullness/annotationsOnExtends?